kogito-timer-springboot/README.md

```diff
@@ -161,11 +254,20 @@ You need to download the job service and start it locally
 java -Dquarkus.http.port=8085 -jar target/jobs-service-{version}-runner.jar
 ```
 
+* After Starting Kogito Web Service you should see a similar Log as follows
+
+<p align="center"><img src="docs/images/kogitoWebServiceLog.png"></p>
+
+
 In case you'd like to run the job service with enabled persistence then start 
 Infinispan server before and then run the job service with following command
 
 ```
 java -Dquarkus.http.port=8085 -Dkogito.jobs-service.persistence=infinispan -jar target/jobs-service-{version}-runner.jar
 ```
 
-in both cases replace `{version}` with actual Kogito version to be used (Job Service is available from 0.6.0)
\ No newline at end of file
+in both cases replace `{version}` with actual Kogito version to be used (Job Service is available from 0.6.0)
+
+* After Starting Infinispan you should see a similar Log as follows
+
+<p align="center"><img src="docs/images/infinispanRunning.png"></p>
```

Comment:
Add after that that the user can redo the timers queries that were described above.

Reply:
fixed @radtriste